src/MenuButton/index.scss

```diff
@@ -27,7 +36,7 @@
 [data-reach-menu-popover] {
   display: block;
   position: absolute;
-  z-index: 2;
+  z-index: 3;
```

Comment:
I have a follow up ticket for this - there is a regression in stacking order

Reply:
Ah yeah, I recall having to do this for other drop downs due to stacking order when they're in dialogs.